packages/controllers/organizations.coffee

```diff
@@ -6,6 +6,13 @@ if Meteor.isClient
     organizations: ->
       if Meteor.userId()
         Organizations.find({createdById: Meteor.userId()})
+    truncated: (description)->
+      description = description.split(' ')
+      wordCount = 50
+      if description.length > wordCount
+        description.slice(0,wordCount).join(' ')+'...'
+      else
+        description.join(' ')
```

Comment:
As a "truncatedDescription" property? Or a method that returns the truncated description?